attachment_s3/models/ir_attachment.py

```diff
@@ -139,6 +139,8 @@ def _store_file_read(self, fname):
     @api.model
     def _store_file_write(self, key, bin_data):
         location = self.env.context.get('storage_location') or self._storage()
+        metadata_dbname = self.env['ir.config_parameter'].sudo().get_param(
+            'ir_attachment.s3.object.metadata') or False
```

Comment:
rename key to attachment_s3.store_db_name_as_metadata

Reply:
Done